### PR TITLE
[wallet] Clarify rpc help message with regard to rounding

### LINKED ISSUE #7331: Amount parser is too strict
{"mvjWntqo6nos9XuskZPyvnWUqR69SqAJsh": 5.4750000000000005, "mugHAcC484RLgqNS8hr1aBk7fhMMaFxuUY": 5.495}

JSONRPC error: Invalid amount

Many JSON serialisers do not give people the freedom to choose exact precision of Numbers.

### PR DESCRIPTION
**Pull request by maflcko**
Clean merge into master and 0.12.

Fixes #7331.

### CODE REVIEW

**Comment by jonasschnelli** [comment]
Does `creatrawtransaction` also requires an update?

**Comment by MarcoFalke** [comment]
Right, it uses `AmountFromValue()` as well. Good catch!

File: src/wallet/rpcwallet.cpp [hunk 1]
```diff
@@ -2180,7 +2179,7 @@ UniValue settxfee(const UniValue& params, bool fHelp)
             "settxfee amount\n"
             "\nSet the transaction fee per kB. Overwrites the paytxfee parameter.\n"
             "\nArguments:\n"
-            "1. amount         (numeric, required) The transaction fee in " + CURRENCY_UNIT + "/kB rounded to the nearest 0.00000001\n"
+            "1. amount         (numeric or sting, required) The transaction fee in " + CURRENCY_UNIT + "/kB\n"
```
Comment (PRabahy) [inline src/wallet/rpcwallet.cpp]: Typo.

Comment (MarcoFalke) [inline src/wallet/rpcwallet.cpp]: Thanks, scheduled for rc2 ;)

**Pull request merged**